extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientRuntimeConfig.java

```diff
@@ -166,6 +166,19 @@ public class InfinispanClientRuntimeConfig {
     @ConfigItem
     Optional<List<String>> sslCiphers;
 
+    /**
+     * Do SSL hostname validation.
+     * Defaults to true.
```

Comment:
This is breaking change, my application failed to start until I set `quarkus.infinispan-client.sni-host-name=localhost`. Please add this to the migration guide, I can't find anything about it in https://infinispan.org/docs/stable/titles/server/server.html and https://quarkus.io/guides/infinispan-client-reference. I take it previously SNI wasn't supported?

Reply:
It's related to this: https://issues.redhat.com/browse/ISPN-15069

Reply:
I will add some information on the ref guide :)

Reply:
@michalvavrik https://github.com/quarkusio/quarkus/pull/36519